packages/block-library/src/cover/shared.js

```diff
@@ -109,3 +117,62 @@ export function getPositionClassName( contentPosition ) {
 
 	return POSITION_CLASSNAMES[ contentPosition ];
 }
+
+function retrieveFastAverageColor() {
+	if ( ! retrieveFastAverageColor.fastAverageColor ) {
+		retrieveFastAverageColor.fastAverageColor = new FastAverageColor();
+	}
+	return retrieveFastAverageColor.fastAverageColor;
+}
+
+/**
+ * getCoverIsDark is a method that specifyies if the cover background is dark or not and
+ * applies the relevant attribute to help ensure that text is visible by default.
```

Comment:
Nit: `getCoverIsDark` no longer applied the attribute, rather it's handled in the block's code. Shall we update this comment?

Reply:
Thanks, fixed.